### ISSUE TITLE
NaN loss during finetuning with AVEX backbone

### ISSUE THREAD

**Issue by ilyassmoummad** [earthspecies/avex#169]
First of all, thank you for releasing both the code and pretrained models, this is extremely valuable for the community.

I encountered an issue when fine-tuning a model loaded via AVEX. Specifically, the training loss becomes NaN during training when using the backbone directly from AVEX.

Here is a simplified version of my setup:
```py
backbone = load_model("esp_aves2_effnetb0_all", device="cuda", return_features_only=True)
classifier = ...
backbone.train()
classifier.train()

# training loop
```

During training, the loss quickly turns into NaN in the first iterations.

As a workaround, I was able to avoid this issue by instantiating a fresh EfficientNet model from ```torchvision``` and copying the weights from the AVEX-loaded backbone into it. This resolves the NaN problem, but I’m unsure why the original AVEX-loaded model does not behave correctly when fine-tuned directly.
